lib/osf-components/addon/components/addon-card/component.ts

```diff
@@ -0,0 +1,58 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+import LegacyProvider from 'ember-osf-web/packages/addons-service/legacy-provider';
+import Provider from 'ember-osf-web/packages/addons-service/provider';
+import AddonsServiceManagerComponent from 'osf-components/components/addons-service/manager/component';
+
+interface Args {
+    addon: LegacyProvider | Provider;
+    manager: AddonsServiceManagerComponent;
+}
+
+const addonLogoMap: Record<string, string> = {
+    aws: '/assets/images/addons/logos/aws.png',
+    bitbucket: '/assets/images/addons/logos/bitbucket.png',
+    boa: '/assets/images/addons/logos/boa_color.png',
+    box: '/assets/images/addons/logos/box.png',
```

Comment:
Since this one will be coming from the services API, we shouldn't need to store this one. That being said, I have no idea how we'd mirage that, so it may just not show properly when we're running locally though mirage.